docs/tutorials/projects/filtered-camera.md

```diff
@@ -7,7 +7,6 @@ image: "/tutorials/filtered-camera-module/viam-figure-preview.png"
 images: ["/tutorials/filtered-camera-module/viam-figure-preview.png"]
 imageAlt: "The wooden Viam figure being detected by a transform camera"
 tags: ["camera", "vision", "detector", "mlmodel", "data"]
-no_list: true
```

Comment:
I happened to notice it was on here while looking for frontmatter examples, and thought it shouldn't be since this tutorial's end doesn't contain anything about getting help in the discord (whereas some do, and that's usually why we set no_list: true on those). I realize it's not related to this tutorial, but it was a nit I figured I could fix here instead of its whole own PR. I can remove if you want?

Reply:
The change to tutorials.html also isn't directly related to this PR but is just a nit fix.